hh-plus-plus.js

```diff
@@ -7053,22 +7053,36 @@ function moduleContestRewards() {
         const savedPanel=contestPanel.innerHTML;
 
         function displayRewardSums(){
-            const contests=$(".contest:not(.is_legendary) .contest_header.ended .personal_rewards .reward_wrap .slot");
+            const contests=$(".contest_header.ended .slot, .contest_header.ended .shards_girl_ico");
```

Comment:
This is selecting the rewards boxes rather than the contests, consider renaming the variable to make it more readable.